samples/tab-approval/nodejs/.env

```diff
@@ -0,0 +1,4 @@
+ClientId = 
+ClientSecret =
+AllowedHosts = 
+TenantId = 
```

Comment:
In C# sample, we have added lots of things in AppSettings but not here:
"AzureAd": {
    "Instance": "https://login.microsoftonline.com/",
    "TenantId": "",
    "MicrosoftAppId": "",
    "MicrosoftAppPassword": "",
    "ApplicationIdURI": "api://{Base_URL}/{MicrosoftAppId}",
    "AuthUrl": "/oauth2/v2.0/token",
    "ValidIssuers": "https://login.microsoftonline.com/tenantId/v2.0,https://sts.windows.net/tenantId/"
  },
  
  Please check once.